proc_dash/app.py

```diff
@@ -50,11 +84,16 @@
                     sort_mode="multi",
                     filter_action="native",
                     page_size=50,
-                    fixed_rows={"headers": True},
+                    # fixed_rows={"headers": True},
                     style_table={"height": "300px", "overflowY": "auto"},
                     style_cell={
                         "fontSize": 13  # accounts for font size inflation by dbc theme
                     },
+                    style_header={
```

Comment:
I don't fully understand what you're fixing here. did `fixed_rows` not achieve what you were looking for? 
If the issue is that column headers sometimes get cut off, I don't think there really is a great way around it (unless you want to waste a lot of whitespace by making columns much wider than the data). Maybe this would help: https://dash.plotly.com/datatable/tooltips#tooltips-in-column-headers

Reply:
So, `fixed_rows={"headers": True}` _did_ achieve the behaviour of keeping the header fixed during vertical scrolling in the datatable, but the issue is that it also caused column widths to change unexpectedly such that some columns where the column name was wider than the data became truncated (where they weren't previously, without `fixed_rows`).

For example, here's what the columns look like _without_ `fixed_rows`: there's more than enough space to accommodate the column names (and columns have equal widths):
<img width="946" alt="normal_header" src="https://github.com/neurobagel/proc_dash/assets/65699389/f318cb5b-1922-442e-90b3-6ed7f3b37b9f">

With `fixed_rows={"headers": True}`, this is what it looks like - notice that the "session" column has been resized to be much narrower than the others, and the column name is also truncated. On my smaller laptop screen, I only see the "s".
<img width="947" alt="fixed_header" src="https://github.com/neurobagel/proc_dash/assets/65699389/4abfada4-b71d-4e49-975a-a1c027b4ef1a">

This appears to be a known bug of `fixed_rows` (see link in issue description). I found the current workaround of specifying the following argument for the DataTable instead of using `fixed_rows`  from a related open issue in Plotly:
```python
  style_header={
    "position": "sticky",
    "top": 0,
  },
```
By fixing the header using CSS styles directly this way, the column widths appear as normal as in the first screenshot, so there is no weird truncation.